### PR TITLE
Push logic down the stack; closes #187

### LINKED ISSUE #187: pull logic out of controllers

### PR DESCRIPTION
**Pull request by EmilyMB**
Added: a few private methods to whitelist params

Moved: create order items method from the controller to the order model

### CODE REVIEW

File: spec/integration/guest_view_spec.rb [hunk 1]
```diff
@@ -107,13 +107,27 @@
...
+      expect(current_path).to eq(root_path)
+    end
   end
 
   def create_item
     image = create(:image)
-    item = create(:item, title: "milk",
-                                     description: "some cheese stuff",
-                                     image_id: image.id)
+    item = create(:item,
```
Comment (KristaANelson) [inline spec/integration/guest_view_spec.rb]: Have you pulled from the most recent master yet? I think may not pass now do to the update validations set on Item.

Comment (EmilyMB) [inline spec/integration/guest_view_spec.rb]: Fixed now.

File: public/assets/manifest-233323c850d1b4c21b6cc024a6893a5b.json [hunk 2]
```diff
@@ -1 +1,5 @@
...
\ No newline at end of file
+<<<<<<< HEAD
```
Comment (KristaANelson) [inline public/assets/manifest-233323c850d1b4c21b6cc024a6893a5b.json]: Merge conflict, I'm guessing we need the HEAD in this case?

Comment (EmilyMB) [inline public/assets/manifest-233323c850d1b4c21b6cc024a6893a5b.json]: I don't know. I haven't read up on asset pipeline...

File: app/controllers/admin_controller.rb [hunk 3]
```diff
@@ -7,4 +7,10 @@ def index
       @orders = Order.sorted.where("status = ?", params[:status])
     end
   end
+
+  private
```
Comment (mikedao) [inline app/controllers/admin_controller.rb:11]: i like!

**Pull request merged**